jwst/saturation/saturation.py

```diff
@@ -22,7 +22,7 @@
 ATOD_LIMIT = 65535.  # Hard DN limit of 16-bit A-to-D converter
 
 
-def flag_saturation(input_model, ref_model, n_pix_grow_sat):
+def flag_saturation(input_model, ref_model, n_pix_grow_sat, use_readpatt):
```

Comment:
Add use_readpatt to parameters in docstring

Reply:
Done!